engine_graphics/src/gl_extensions.cpp

```diff
@@ -0,0 +1,1011 @@
+///////////////////////////////////////////////////////////////////////////////
+// glExtension.cpp
+// ===============
+// OpenGL extension helper
+// If an ARB extension is included in OpenGL core, it uses the function poiters
+// and tokens without ARB suffix.
+// NOTE: In order to get valid OpenGL extensions on Windows system, HDC must be
+// passed to wglGetExtensionsStringARB() function. The size of HDC in 64bit
+// Windows is 8 bytes.
+//
+// extensions
+// ==========
+// GL_ARB_framebuffer_object
+// GL_ARB_debug_output
+// GL_ARB_direct_state_access
+// GL_ARB_multisample
+// GL_ARB_multitexture
+// GL_ARB_pixel_buffer_objects, GL_ARB_vertex_buffer_object
+// GL_ARB_shader_objects, GL_ARB_vertex_program, GL_ARB_fragment_program, GL_ARB_vertex_shader, GL_ARB_fragment_shader
+// GL_ARB_sync
+// GL_ARB_vertex_array_object
+// WGL_ARB_extensions_string
+// WGL_ARB_pixel_format
+// WGL_ARB_create_context
+// WGL_EXT_swap_control
+//
+//  AUTHOR: Song Ho Ahn (song.ahn@gmail.com)
+// CREATED: 2013-03-05
+// UPDATED: 2018-08-09
+///////////////////////////////////////////////////////////////////////////////
+
+#include <iostream>
+#include <algorithm>
+#include <core/gl_extensions.hpp>
+
+
+#ifdef _WIN32 //===============================================================
```

Comment:
Remove windows related stuff.

Reply:
The engine supposed to be cross-platform, so this stays.